### ISSUE TITLE
Test failure (`test_zzphase`)

### ISSUE THREAD

**Issue by cqc-alec** [CQCL/pytket-quantinuum#25]
Running this test on current `develop`:

```
    @pytest.mark.skipif(skip_remote_tests, reason=REASON)
    @pytest.mark.parametrize(
        "authenticated_quum_backend", [{"device_name": "H1-1SC"}], indirect=True
    )
    def test_zzphase(
        authenticated_quum_backend: QuantinuumBackend,
    ) -> None:
        backend = authenticated_quum_backend
        c = Circuit(2, 2, "test rzz")
        c.H(0)
        c.CX(0, 1)
        c.Rz(0.3, 0)
        c.CY(0, 1)
        c.ZZPhase(0.1, 1, 0)
        c.measure_all()
        c0 = backend.get_compiled_circuit(c, 0)
    
>       assert c0.n_gates_of_type(OpType.ZZPhase) > 0
E       assert 0 > 0
E        +  where 0 = <bound method PyCapsule.n_gates_of_type of [Rz(1) q[0]; Rz(1) q[1]; PhasedX(0.5, 0.5) q[0]; PhasedX(0.5, 1) q[1]; Rz(3...0) q[1]; Rz(1.5) q[0]; PhasedX(0.5, 1.5) q[1]; Measure q[1] --> c[1]; PhasedX(0.5, 1.5) q[0]; Measure q[0] --> c[0]; ]>(<OpType.ZZPhase: 68>)
E        +    where <bound method PyCapsule.n_gates_of_type of [Rz(1) q[0]; Rz(1) q[1]; PhasedX(0.5, 0.5) q[0]; PhasedX(0.5, 1) q[1]; Rz(3...0) q[1]; Rz(1.5) q[0]; PhasedX(0.5, 1.5) q[1]; Measure q[1] --> c[1]; PhasedX(0.5, 1.5) q[0]; Measure q[0] --> c[0]; ]> = [Rz(1) q[0]; Rz(1) q[1]; PhasedX(0.5, 0.5) q[0]; PhasedX(0.5, 1) q[1]; Rz(3.5) q[0]; PhasedX(0.5, 0) q[1]; PhasedX(0.5... 0) q[1]; Rz(1.5) q[0]; PhasedX(0.5, 1.5) q[1]; Measure q[1] --> c[1]; PhasedX(0.5, 1.5) q[0]; Measure q[0] --> c[0]; ].n_gates_of_type
E        +    and   <OpType.ZZPhase: 68> = OpType.ZZPhase

backend_test.py:547: AssertionError
```

**Comment by lmondada** [comment]
> Should be solved by [CQCL/tket#492](https://github.com/CQCL/tket/pull/492).

After looking into it, I actually don't think these are related. It might be a change in the gatesets advertised by the quantinuum API.

**Comment by cqc-alec** [comment]
> > Should be solved by [CQCL/tket#492](https://github.com/CQCL/tket/pull/492).
> 
> After looking into it, I actually don't think these are related. It might be a change in the gatesets advertised by the quantinuum API.

Yup. I will check.

**Comment by cqc-alec** [comment]
Yes, this is the advertised gate set currently:
```
{<OpType.ZZMax: 65>, <OpType.Rz: 33>, <OpType.Barrier: 6>, <OpType.WASM: 12>, <OpType.SetBits: 13>, <OpType.CopyBits: 14>, <OpType.RangePredicate: 15>, <OpType.ExplicitPredicate: 16>, <OpType.ExplicitModifier: 17>, <OpType.MultiBit: 18>, <OpType.ClassicalExpBox: 87>, <OpType.Measure: 58>, <OpType.Reset: 60>, <OpType.PhasedX: 63>}

```
We just need to fix the test.
